modules/router-store/src/serializers/full_serializer.ts

```diff
@@ -6,7 +6,10 @@ export interface SerializedRouterStateSnapshot extends BaseRouterStoreState {
   url: string;
 }
 
-export class DefaultRouterStateSerializer
+/**
+ * @deprecated The FullRouterStateSerializer cannot be used when serializability runtime checks are enabled. Use the {@link MinimalRouterStateSerializer} or a {@link https://ngrx.io/guide/router-store/configuration#custom-router-state-serializer custom serializer} instead.
```

Comment:
I'm not sure if we should deprecate this.
It's recommended to use the minimal serializer, but some (a lot?) applications have serializability disabled, making this serializer valid.

Thoughts?